Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# 2nd Balance Tweaks

## Changelog:
- With over 50% Threat all Defensive Pacts of other religion groups band together if you attack one of them.


# How to test:
Observe a few games and tell your opinion. It should make the game less aggresive.
